spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'Viewing peeps' do
+  scenario 'A user can see peeps in reverse chronological order' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    connection.exec("INSERT INTO peeps VALUES(1, 'Good morning');")
+    connection.exec("INSERT INTO peeps VALUES(2, 'Good afternoon');")
+    connection.exec("INSERT INTO peeps VALUES(3, 'Good night');")
+
+    visit('/peeps')
+
+    expect(page).to have_content "Good night"
```

Comment:
I think if you add a 'class' to the list items:
```<li class='peep'><%= peep.text %></li>```
you can check specific list items using:
```expect(first('.list_item_class')).to have_content 'content text'```